### PR TITLE
INVS-1978 - Threat indicators format

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds file format examples to this beta article:
https://help.sumologic.com/docs/platform-services/threat-intelligence-indicators/

Issue: [INVS-1978](https://sumologic.atlassian.net/browse/INVS-1978)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

### CODE REVIEW

File: docs/platform-services/threat-intelligence-indicators.md [hunk 1]
```diff
@@ -248,4 +190,288 @@ Parameters:
...
+      }
+    }
+  ]
+}
+```
+#### Required attributes
+
+For information about the attributes to use, see ["Indicator" in the STIX 2.1 specification](https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_muftrcpnf89v), and the [uploadNormalizedIndicators API](https://api.sumologic.com/docs/#operation/uploadNormalizedIndicators) in the [threatIntelIngest](https://api.sumologic.com/docs/#tag/threatIntelIngest) resource. 
+
+The following attributes are required:
```
Comment (mvbrock) [inline docs/platform-services/threat-intelligence-indicators.md:236]: Somehow Github's markdown rendering put all these bullets on the same line. But I'm not sure if users read the documentation on Github or if this is getting rendered somewhere else.

File: docs/platform-services/threat-intelligence-indicators.md [hunk 2]
```diff
@@ -68,75 +73,12 @@ You can also add threat intelligence indicators using the API or a collector. Se
 1. In Sumo Logic, go to **Manage Data > Logs > Threat Intelligence**.
 1. Click **Add Indicators**. The dialog displays. <br/><img src={useBaseUrl('img/platform-services/threat-intelligence-add-indicators.png')} alt="Add threat intelligence indicators" style={{border: '1px solid gray'}} width="500" />
```
Comment (mvbrock) [inline docs/platform-services/threat-intelligence-indicators.md:74]: Similarly, I'm not sure what will be rendering the markdown, but Github needs the `alt text` format. But if it's some other renderer then maybe this works.

**Review by mvbrock — APPROVED** [review]
A couple nitpicks but I'm not sure if users read these docs through Github or somewhere else. Looks good!

File: docs/platform-services/threat-intelligence-indicators.md (hunk 2 above)
Comment (jpipkin1) [reply docs/platform-services/threat-intelligence-indicators.md:74]: Rendering is done by Docusaurus. The above renders as:

Screenshot 2024-01-23 at 9 43 48 AM

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**